src/ConnectedMode/Suppressions/SuppressionIssueStoreUpdater.cs

```diff
@@ -30,16 +37,121 @@ internal interface ISuppressionIssueStoreUpdater
         /// <summary>
         /// Fetches all available suppressions from the server and updates the server issues store
         /// </summary>
-        void FetchAllServerSuppressions();
+        Task UpdateAllServerSuppressionsAsync();
```

Comment:
I changed the methods to be `async` - we're making network calls, so they should be `async`.